index.php

```diff
@@ -144,6 +147,8 @@
 		$_cmds[] = makeLink(__('settings'), makeAction('settings', '', ''));
 		$_cmds[] = makeLink(__('logout'), makeAction($hyphaQuery, 'logout', ''));
 	}
+	$_cmds[] = '<span onclick="toggleInfoButtons(this)" class="toggleinfo">?</span>';
```

Comment:
IMHO we should only offer this option if it persists across navigation to different pages. If on every url change the icons turn back on, this is probably only annoying.